adapter/src/main/java/org/jvalue/ods/adapterservice/config/RabbitConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package org.jvalue.ods.adapterservice.config;
+
+import org.springframework.amqp.core.Queue;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RabbitConfiguration {
+    public static final String DATA_IMPORT_QUEUE = "dataImportQueue";
```

Comment:
In documentation of RabbitMQ queues are defined in format `asd_def`, so no lower camel case.. What do you think? I would name this one `data-import` or `data_import`

Reply:
Agree!